.builders/scripts/repair_wheels.py

```diff
@@ -97,6 +100,25 @@ def __str__(self):
         ]) + '.whl'
 
 
+def check_unacceptable_files(
+    wheel: Path,
+    bypass_prefixes: list[str],
+    invalid_file_patterns: list[str],
+):
+    """Check if a wheel contains any unacceptable files and exit if found."""
+    if any(wheel.name.startswith(pkg_prefix) for pkg_prefix in bypass_prefixes):
+        return
```

Comment:
Since (I guess) we aim at removing the tolerance, I would add some verbosity about it (warning or so). What do you think?

Reply:
Much appreciated!